cmd/amp/platform_manager.go

```diff
@@ -615,6 +616,51 @@ func (s *ampManager) close() {
 	//s.docker.Close()
 }
 
+func (s *ampManager) getTheme() string {
+	homedir, err := homedir.Dir()
+	if err != nil {
+		return ""
+	}
+	file, err := os.Open(path.Join(homedir, ".amp.yaml"))
+	if err != nil {
+		return ""
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		param := strings.Split(scanner.Text(), ":")
+		if strings.Trim(param[0], " ") == "cmd-theme" {
+			if len(param) < 2 {
+				return ""
+			} else {
+				return strings.Trim(param[1], " ")
+			}
+		}
+	}
+	return ""
+}
+
+func (s *ampManager) setColors() {
+	theme := s.getTheme()
+	if theme == "white" {
```

Comment:
I think the name of the theme should not be based on one of the color of the theme, but should match the terminal theme on which it displays best.
So here white should be black.